typescript/web/src/components/dataset-images-list/image-card.tsx

```diff
@@ -58,65 +43,110 @@ const DeleteButton = () => {
   );
   return (
     <IconButton
-      icon={<TrashIcon />}
+      _hover={{ bgColor: "rgba(0, 0, 0, .2)" }}
+      _active={{ bgColor: "gray.600" }}
+      color="white"
+      icon={<TrashIcon boxSize={4} />}
       aria-label="delete image"
       isRound
       size="sm"
       onClick={handleClick}
+      variant="ghost"
     />
   );
 };
 
-const Header = () => (
-  <Flex justifyContent="space-between" w="100%" alignItems="center">
-    <NameHeading />
-    <DeleteButton />
-  </Flex>
-);
-
-const Thumbnail = () => {
+const ImageContent = () => {
   const { name, thumbnail } = useImageCard();
   return (
     <ImageWithFallback
-      background={mode("gray.100", "gray.800")}
+      borderRadius={8}
       alt={name}
       src={thumbnail ?? undefined}
       loadingFallback={<Skeleton height="100%" width="100%" />}
       errorFallback={<EmptyStateImageNotFound />}
-      objectFit="contain"
+      objectFit="cover"
       h="208px"
       w="full"
-      flexGrow={0}
-      flexShrink={0}
     />
   );
 };
 
-const Body = () => (
-  <VStack
-    maxW="486px"
-    p={4}
-    background={mode("white", "gray.700")}
-    rounded={8}
-    height="270px"
-    justifyContent="space-between"
+const OverlayTopRow = () => (
+  <HStack
+    visibility="hidden"
+    justifyContent="flex-end"
+    w="100%"
+    sx={{
+      ".imageCard:hover &": {
+        visibility: "visible",
+      },
+    }}
```

Comment:
You can use the Chakra `role="group"` and `_groupHover` for this specific case.
https://github.com/chakra-ui/chakra-ui/discussions/2386#discussioncomment-118607

If it still doesn't work with it, we should bind the correct events of ImageCard to the `setOn` or `setTrue` return properties of the `useBoolean` hook and forward the boolean value to the component.